src/services/codefixes/helpers.ts

```diff
@@ -381,14 +391,18 @@ namespace ts.codefix {
             createStubbedMethodBody(quotePreference));
     }
 
-    function createStubbedMethodBody(quotePreference: QuotePreference): Block {
+    function createStubbedMethodBody(quotePreference: QuotePreference) {
+        return createStubbedBody("Method not implemented.", quotePreference);

```

Comment:
shouldn't this be a message in diagnosticMessages? I feel like people will want localised versions of it.

Reply:
Good question. I think you are right, I don't know why this message wasn't added to the Diagnostics before. Anyway I'll add the following messages to Diagnostics list

1. `"Method not implemented."`
2. `"Function not implemented."`